### ISSUE TITLE
Large Scale Distributed Collaborative Unlabeled Motion Planning with Graph Policy Gradients. (arXiv:2102.06284v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#3638]
Large Scale Distributed Collaborative Unlabeled Motion Planning with Graph Policy Gradients. (arXiv:2102.06284v1 [cs.RO])<br>
https://ift.tt/3jOSqqm<br>
<p>In this paper, we present a learning method to solve the unlabelled motion problem with motion constraints and space constraints in 2D space for a large number of robots. To solve the problem of arbitrary dynamics and constraints we propose formulating the problem as a multi-agent problem. We are able to demonstrate the scalability of our methods for a large number of robots by employing a graph neural network (GNN) to parameterize policies for the robots. The GNN reduces the dimensionality of the problem by learning filters that aggregate information among robots locally, similar to how a convolutional neural network is able to learn local features in an image. Additionally, by employing a GNN we are also able to overcome the computational overhead of training policies for a large number of robots by first training graph filters for a small number of robots followed by zero-shot policy transfer to a larger number of robots. We demonstrate the effectiveness of our framework through various simulations.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
